content/administration/upgrade/faq.rst

```diff
@@ -107,26 +119,37 @@ Open the following link to get :doc:`more information about how to change your h
 
 .. _upgrade-faq/release-notes:
 
+The Upgrade Report
+==================
+
+When an upgrade request completes successfully (test or production), you receive an email
+notification about it that includes an 'Upgrade Report'. This report is also sent to you via the
+Discuss app.
```

Comment:
- to the line (line 128) is to write a new paragraph? Then add a blank line between the paragraphs
```suggestion
Discuss app.

```